src/containers/RepositoriesDetailContainer/RepositoriesDetailDuck.js

```diff
@@ -0,0 +1,75 @@
+import ApiUtils from '../../utils/ApiUtils.js';
+
+// Actions
+const LOAD = 'react-redux-starter-app/repositoriesDetail/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositoriesDetail/LOAD_SUCCESS';
+const LOAD_ERROR = 'react-redux-starter-app/repositoriesDetail/LOAD_ERROR';
+
+const initialState = {
+  isLoading: false,
+  repos: undefined
+};
+
+// Reducers
+const REDUCERS = {
+  [LOAD]: (state, action) => ({
+    ...state,
+    isLoading: true
+  }),
+  [LOAD_SUCCESS]: (state, action) => ({
+    ...state,
+    isLoading: false,
+    repos: action.data.map(repo => ({
+      author: repo.owner.login,
+      avatar: repo.owner.avatar_url,
+      description: repo.description,
+      id: repo.id,
+      name: repo.name,
+      starsCount: repo.stargazers_count,
+      url: repo.html_url
+    }))
+  }),
+  [LOAD_ERROR]: (state, action) => ({
+    ...state,
+    isLoading: false,
+    error: action.error
+  })
+};
+
+export default function reducer (state = initialState, action = {}) {
+  const handler = REDUCERS[action.type];
+  return handler ? handler(state, action) : state;
+};
+
+// Action Creators
+export function load () {
+  return (dispatch, getState) => {
+    // Already loading?
```

Comment:
As a starter kit, especially for people getting used to react & redux. I think it's handy to have these comments?